.github/workflows/pnp-build-input-api.yml

```diff
@@ -209,6 +214,13 @@ jobs:
         with:
           service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }}
 
+      - name: Restore AsyncAPI schema from cache
+        if: ${{ inputs.include-asyncapi-schema == true }}
+        uses: actions/cache@v3
+        with:
+          path: schemas/v1/asyncapi.yaml
```

Comment:
I see we have many paths defined as secrets. Like path sln, path to docker file. I don't know if we need hide schema path the same way though. Maybe check with platform team?